setup.sh

```diff
@@ -0,0 +1,382 @@
+#!/bin/bash
+set -e
+
+# list of colors
+BOLD=$(tput bold)
+RED="\\033[1;31m"
+BLUE="\\033[1;34m"
+GREEN="\\033[1;32m"
+YELLOW="\\033[1;33m"
+NORMAL="\\033[0;39m"
+RT="\r\n"
+
+CURRENT_OS=Unknown
+OS_IS_SUPPORTED=0
+DOCKER_COMPOSE_BIN=/usr/local/bin/docker-compose
+
+# Output a text with the selected color (reinit to normal at the end)
+write() {
+  echo -e " $1$2" "$NORMAL" >&2
```

Comment:
oh...
honestly, I didn't even notice it when I copy-pasted it from a script by @gandbox .
Took it out now.